src/server.js

```diff
@@ -215,5 +224,91 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_ADDRESS}/geocode`,
+  async (request, response) => {
+    let address = request.params[URL_PARAM_ADDRESS];
+
+    let geocodeRequest =
```

Comment:
Nit: `const`

Reply:
There's also at least 1 other place in the code, where `let` could be replaced with `const`.

Reply:
Fixed.